lib/logstash/inputs/rabbitmq/march_hare.rb

```diff
@@ -100,6 +102,9 @@ def setup
 
       # exchange binding is optional for the input
       if @exchange
+        @ch.exchange(@exchange,
```

Comment:
@ekimekim would you mind removing this declaration so we can merge this, then maybe open a new pr on allowing exchange declaration?